apiv1/models/CourseModel.js

```diff
@@ -4,7 +4,17 @@ let Schema = mongoose.Schema;
 let LessonSchema = new Schema({
 	'name': { type: String, required: true },
 	'prompt': String,
-	'code': { type: String, required: true }
+	'code': { type: String, required: true },
+    'settings': {
```

Comment:
I think it would make sense to include your light settings since we're merging that over.